app/celery/process_delivery_status_result_tasks.py

```diff
@@ -81,7 +81,7 @@ def process_delivery_status(self, event: CeleryEvent) -> bool:
     try:
         # calculate pricing
         current_app.logger.info(
-            "Notification ID (%s) - Calculate Pricing: %s and notification_status: %s with number_of_message_parts",
+            "Notification ID (%s) - Calculate Pricing: %s and notification_status: %s with number_of_message_parts: %s",
```

Comment:
Just a note, %s is also fine here for logging. It will first call int() on the argument and then call str().  Although %d does make it clear what type of value the argument is.

Reply:
We use %s for all. Please review [our logging policy](https://github.com/department-of-veterans-affairs/vanotify-team/blob/master/Engineering/logging_policy.md#style).
We will only use `%s`.